modules/serverless-ossm-traffic-isolation-securing-the-service-mesh.adoc

```diff
@@ -0,0 +1,263 @@
+        - source:
+            namespaces: [ "knative-eventing" ]
+            principals: [ "cluster.local/ns/knative-eventing/sa/kafka-controller" ]
+      to:
+        - operation:
+            methods: [ "GET" ]
+---
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: allow-traffic-to-activator
+  namespace: knative-serving
+spec:
+  selector:
+    matchLabels:
+      app: activator
+  action: ALLOW
+  rules:
+    - from:
+        - source:
+            namespaces: [ "knative-serving", "istio-system" ]
+---
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: allow-traffic-to-autoscaler
+  namespace: knative-serving
+spec:
+  selector:
+    matchLabels:
+      app: autoscaler
+  action: ALLOW
+  rules:
+    - from:
+        - source:
+            namespaces: [ "knative-serving" ]
+----
++
+These policies restrict the access rules for the network communication between {ServerlessProductShortName} system components. Specifically, they enforce the following rules:
++
+--
+* Deny all traffic that is not explicitly allowed in the `knative-serving` and `knative-eventing` namespaces
+* Allow traffic from the `istio-system` and `knative-serving` namespaces to activator
+* Allow traffic from the `knative-serving` namespace to autoscaler
+* Allow health probes for Apache Kafka components in the `knative-eventing` namespace
+* Allow internal traffic for channel-based brokers in the `knative-eventing` namespace
+--
+
+. Apply the authorization policy configuration:
++
+[source,terminal]
+----
+$ oc apply -f knative-default-authz-policies.yaml
+----
+
+. Define which OpenShift projects can communicate with each other. For this communication, every OpenShift project of a tenant requires the following:
++
+--
+* One `AuthorizationPolicy` object limiting directly incoming traffic to the tenant's project
+* One `AuthorizationPolicy` object limiting incoming traffic using the activator component of {ServerlessProductShortName} that runs in the `knative-serving` project
+* One `AuthorizationPolicy` object allowing Kubernetes to call `PreStopHooks` on Knative Services
+--
++
+Instead of creating these policies manually, install the `helm` utility and create the necessary resources for each tenant:
++
+.Installing the `helm` utility
+[source,terminal]
+----
+$ helm repo add openshift-helm-charts https://charts.openshift.io/
+----
++
+.Creating example configuration for `team alpha`
+[source,terminal,subs="attributes+"]
+----
+$ helm template openshift-helm-charts/redhat-knative-istio-authz --version {ServerlessProductVersion} --set "name=team-alpha" --set "namespaces={team-alpha-1,team-alpha-2}" > team-alpha.yaml
+----
++
+.Creating example configuration for `team bravo`
+[source,terminal,subs="attributes+"]
+----
+$ helm template openshift-helm-charts/redhat-knative-istio-authz --version 1.31.0 --set "name=team-bravo" --set "namespaces={team-bravo-1,team-bravo-2}" > team-bravo.yaml
+----
```

Comment:
For a follow-up: Using block titles to describe (what seems like) sequential commands/steps is a bit non-standard. I'd suggest breaking these each into actual substeps.